apollo-core/src/main/java/com/ctrip/framework/foundation/internals/provider/DefaultApplicationProvider.java

```diff
@@ -86,7 +86,15 @@ private void initAppId() {
       return;
     }
 
-    // 2. Try to get app id from app.properties.
+    //2. Try to get app id from OS environment variable
+    m_appId = System.getenv("APP_ID");
+    if (!Utils.isBlank(m_appId)) {
+      m_appId = m_appId.trim();
+      logger.info("App ID is set to {} by APP_ID env from OS environment variable", m_appId);
```

Comment:
one minor typo fix:

`App ID is set to {} by APP_ID property from OS environment variable`